qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -53,6 +53,9 @@
 # Pre-application setup
 plugins.setup_library_paths()
 
+from mantid.kernel import version_str
```

Comment:
How  does this find the correct properties file? There is a line at the start of `def main()` that sets `MANTIDPATH` correctly but that won't have happened yet.

Reply:
Ignore the above comment regarding the properties file - I'd forgotten we handle it internal within `mantid`.

There is unfortunately still an issue with having the `import` at this level: the workbench message in window doesn't catch the startup message
```FrameworkManager-[Notice] Welcome to Mantid 3.13.20180815.2055
FrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid```

and instead it goes to the console only. It's basically a chicken and the egg problem because as soon as you `import mantid` then the `FrameworkManager` starts and spits out the start up message but for the workbench to catch them it must have created and connected the `MessageDisplay` widget. This is why the `mantid` import was carefully placed in its current position: https://github.com/mantidproject/mantid/blob/7eb39b1bd4878371ac67d31905a68282f446707a/qt/applications/workbench/workbench/app/mainwindow.py#L442.

This is also the reason I was keen to change the behaviour of `import mantid` a while back so that it would not have any side effects and we could do this sensibly.